communication/moderation.md

```diff
@@ -244,3 +257,4 @@ guidelines:
 [Steering Committee]: https://git.k8s.io/steering
 [moderator request]: https://github.com/kubernetes/community/issues/new/choose
 [moderator template]: https://git.k8s.io/community/.github/ISSUE_TEMPLATE/moderator_application.md
+[filing an issue]: https://github.com/kubernetes/community/issues/new/choose
```

Comment:
I think this is what @mrbobbytables was asking for
```suggestion
[filing an issue]: https://github.com/kubernetes/community/issues/new?assignees=&labels=area%2Fcommunity-management%2C+sig%2Fcontributor-experience&template=moderator_application.md&title=REQUEST%3A+New+moderator+for+%3Cyour-GH-handle%3E+of+%3Ck8s+property%3E
```

Reply:
It was^^;;;